src/assets/css/librarybrowser.css

```diff
@@ -443,24 +487,25 @@
 }
 
 .detailImageContainer {
-    margin: 1.25em 0;
-    position: sticky;
-    top: 15%;
-    float: left;
-    width: 22.786458333333332vw;
+    width: 280px;
+    margin-right: 2em;
+    flex-shrink: 0;
```

Comment:
It looks like you didn't keep the changes from the new item details layout, which will definitely break that page. We need to go through this and make sure to remove those changes.